german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Es gibt "Gambeson" im Deutschen definitiv, aber ich halte es nicht für "geläufig" für "Fachjargon" der Rollenspiel-/Mittelalterfan- und Hobbyhistorikerszene (wo wir wahrscheinlich alle einem gewissen Filterblaseneffekt unterworfen sind). Normalen Menschen kommt sowas meiner Meinung nach vielleicht einmal im Kreuzworträtsel oder als Frage in der zweiten Hälfte von "Wer wird Millionär?" vor.

Es ist natürlich auch immer eine Frage, wer das Publikum ist. Aber manchmal habe ich Angst, dass man sich mit so einem elitären Sprachgebrauch eher unzugänglich für Leute macht, die vielleicht Spaß an dem Spiel hätten, ohne gleich in die Rollenspiel-In-Group eintreten zu müssen.